<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTagDialog</class>
 <widget class="QDialog" name="CreateTagDialog">
  <property name="windowTitle">
   <string>创建标签</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>新建标签</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutName">
     <item>
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>标签名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTagName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutImage">
     <item>
      <widget class="QLabel" name="labelImage">
       <property name="text">
        <string>标签图片：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelImagePreview">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSelectImage">
       <property name="text">
        <string>选择图片...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutButtons">
     <item>
      <widget class="QPushButton" name="buttonBoxOk">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBoxCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 